mysql/README.md

```diff
@@ -1,59 +1,143 @@
 # Overview
 
-Connect MySQL to Datadog in order to:
+The Datadog Agent can collect many metrics from MySQL databases, including those for:
 
-* Visualize your database performance
-* Correlate the performance of MySQL with the rest of your applications
+* Query throughput
+* Query performance (average query run time, slow queries, etc)
+* Connections (currently open connections, aborted connections, errors, etc)
+* InnoDB (buffer pool metrics, etc)
 
-# Installation
+And many more. You can also invent your own metrics using custom SQL queries.
```

Comment:
"SQL queries can be provided to generate custom metrics."

Whose SQL queries? Who provides them? (passive voice)

Unless we want to strictly avoid "you"/"your" in marketing pages (I don't see why we would need to), how about:

"Provide your own SQL queries to create custom metrics."

What's wrong with invent? It's more emotive than create or generate, though maybe it doesn't signal transmission/publishing (to Datadog) as much.